### PR TITLE
fix/LP-190-Fix-Community-View

### PR DESCRIPTION
**Pull request by irv205**
Before:
image

After:
image

- Created a Sealed class CommunityCardType object that enumerates the different types of available cards.
- Created a Community class that contains an icon and a card type, replacing the CommunityCardDataModel class in the first function.
- Replaced size constants with a sizes object that contains the different necessary sizes.
- Created a cardDescriptions object that maps different card types to their descriptions.
- Used remember to cache cardDescriptions.
- Renamed the navigationToCommnityWall variable to navigationToCommunityWall.
- Deleted the cardPadding20 constant.
- Removed some unnecessary padding modifiers in Column and Text.
- Replaced GridCells.Fixed(2) with GridCells.Adaptive(minSize = sizes.extraSize124).
- Changed the name of the itemCard variable to community for clarity.
- Removed the TODO() block for the CommunityCardType.DoItYourself, CommunityCardType.Events, and CommunityCardType.Services options.

### CODE REVIEW

File: feature/community/src/main/java/com/iteneum/community/presentation/CommunityView.kt [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+It takes in two lambda parameters that are called when corresponding buttons are clicked,
+namely navigationToCommunityWall and navigationToAmenities. The function displays a grid
+of community cards, each containing an icon and a description, which when clicked calls
+the appropriate lambda function to navigate to different parts of the application.
+The function uses Jetpack Compose to create a responsive UI.
+
+* @author Irving Gonzalez
+ **/
+@Composable
+fun CommunityView(
```
Comment (melissaquezada94) [inline feature/community/src/main/java/com/iteneum/community/presentation/CommunityView.kt:35]: Please add the parameters of the function in the documentation

File: feature/community/src/main/java/com/iteneum/community/presentation/CommunityView.kt [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+                    .padding(sizes.smallerSize),
+                    icon = itemCard.icon,
+                    description = stringResource(
+                        id = cardDescriptions[itemCard.cardType] ?: R.string.default_card_description
+                    ),
+                    onCardClicked = {
+                        when(itemCard.cardType){
+                            CommunityCardType.Amenities -> {navigationToAmenities()}
+                            CommunityCardType.CommunityWall -> {navigationToCommunityWall()}
+                            CommunityCardType.DoItYourself -> { TODO() }
```
Comment (melissaquezada94) [inline feature/community/src/main/java/com/iteneum/community/presentation/CommunityView.kt]: Please add a message of what needs to do in the TODO comment

File: feature/community/src/main/java/com/iteneum/community/presentation/CommunityView.kt (hunk 1 above)
Comment (irv205) [reply feature/community/src/main/java/com/iteneum/community/presentation/CommunityView.kt:35]: done

File: feature/community/src/main/java/com/iteneum/community/presentation/CommunityView.kt (hunk 2 above)
Comment (irv205) [reply feature/community/src/main/java/com/iteneum/community/presentation/CommunityView.kt]: done

File: feature/community/src/main/java/com/iteneum/community/presentation/CommunityView.kt [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+                            CommunityCardType.Services -> {
+                                /* TODO() This view does not exist, as soon as
+                                    this view is defined the navigation of this section is added */
+                            }
+                        }
+                    })
+            }
+        }
+    }
+}
```
Comment (carlosph816) [inline feature/community/src/main/java/com/iteneum/community/presentation/CommunityView.kt:112]: Can you reformat the code please?

Comment (irv205) [reply feature/community/src/main/java/com/iteneum/community/presentation/CommunityView.kt:112]: done

**Review by melissaquezada94 — APPROVED** [review]
well done

**Pull request merged**